service/src/main/scala/com/yahoo/maha/service/utils/MahaRequestLogUtils.scala

```diff
@@ -164,6 +164,7 @@ case class MahaRequestLogHelper(mahaRequestContext: MahaRequestContext, mahaRequ
     val drivingQuery = queryPipeline.queryChain.drivingQuery
     val model = queryPipeline.queryChain.drivingQuery.queryContext.requestModel
     val factBestCandidateOption = queryPipeline.factBestCandidate
+    val engine = MahaRequestProto.Engine.Hive
```

Comment:
we can not hardcode the engine to hive, we need to read the engine from drivingQuery from query pipeline